client/ayon_resolve/plugins/create/create_shot_clip.py

```diff
@@ -48,42 +161,22 @@ def header_label(text):
                     ),
                     default=True),
 
-            # renameHierarchy
+            # export outputs
             UILabelDef(
-                label=header_label("Shot Hierarchy And Rename Settings")
-            ),
-            TextDef(
-                "hierarchy",
-                label="Shot Parent Hierarchy",
-                tooltip="Parents folder for shot root folder, "
-                        "Template filled with *Hierarchy Data* section",
-                default=presets.get("hierarchy", "{folder}/{sequence}"),
-            ),
-            BoolDef(
-                "clipRename",
-                label="Rename clips",
-                tooltip="Renaming selected clips on fly",
-                default=presets.get("clipRename", False),
-            ),
-            TextDef(
-                "clipName",
-                label="Clip Name Template",
-                tooltip="template for creating shot names, used for "
-                        "renaming (use rename: on)",
-                default=presets.get("clipName", "{sequence}{shot}"),
-            ),
-            NumberDef(
-                "countFrom",
-                label="Count sequence from",
-                tooltip="Set where the sequence number starts from",
-                default=presets.get("countFrom", 10),
-            ),
-            NumberDef(
-                "countSteps",
-                label="Stepping number",
-                tooltip="What number is adding every new step",
-                default=presets.get("countSteps", 10),
+                label=header_label("Additional Export(s)")
             ),
+            BoolDef("export_plate",
+                    label="Plate",
+                    tooltip="Export Plate output(s)",
+                    default=False),
+            BoolDef("export_review",
+                    label="Review",
+                    tooltip="Export Review output(s)",
+                    default=False),
+            BoolDef("export_audio",
+                    label="Audio",
+                    tooltip="Export Audio output(s)",
+                    default=False),
```

Comment:
I suggest we remove the section related to Traypublisher editorial workflow. In Hiero/Resolve/Flame, we can address this by using a combination of `productType`, `reviewTrack`, and `audio` fields. Currently, we need to decide how to handle the `take` type in the `productType` enum field, as it's not yet implemented.

Reply:
@jakubjezek001 sounds good to me ! Would you mind to confirm the expected UX please ?
Just to be sure I got it right. (For now I replicated the exact same behavior then current publisher.)

![image](https://github.com/user-attachments/assets/a13d7f4c-3d20-4139-a984-fd1ec9110928)

* I'm not sure what **Use Review Track** is needed for actually and couldn't really figure out from the code
* **Renaming** seems to be applied only to the shot name but not the resulting plate product(s) name which remain `{product_type}{variant}` (e.g. plate_VideoTrack)
* When creating a timeline from movie clips, **Include Audio** did not seem to have any effect. 
_So I've implemented a new plugin to create "audio" product from the clip source range (no handles) but that might not be what's needed._

Reply:
Here are some notes:

![image](https://github.com/user-attachments/assets/9d1880a4-fbb3-46d4-8bbb-f24818dd6998)

> I'm not sure what Use Review Track is needed for actually and couldn't really figure out from the code

From the image above you can see that additional track could be added to the timeline and that can be then pointed to the **Use Review Track** field. 

> Renaming feature seems to be applied only to the shot name but not the resulting plate product(s) name which remain {product_type}{variant} (e.g. plate_VideoTrack), should we re-label as "Rename Shot" ?

Yes, that's correct for now. It works just as you described. The product name is created by combining the Product Variant field and Product Type like this: `{product_type}{Product_variant}`. We're not using product name presets at the moment because we're still deciding how to handle tasks in the future. We might need to introduce a specific task for plate publishing, as we've seen several client requests for this in the past.

> When creating a timeline from movie clips, Include Audio did not seem to have any effect. Meaning the extracted plate got audio whatever it's checked or not.
So I've implemented a new plugin to create "audio" product from the clip source range (no handles) but that might not be what's needed.

To be honest I am not sure if it ever worked. This had been migrated from Hiero but since noone was ever reporting it it might just slip our attantion. Correct behaviour should be following:

1. Include audio toggled on
2. Additional `audioMain` product instances per clip should be created
3. Publishing process should be coverting whole timeline (included audio tracks) into OTIO and then other Core plugin should use FFMPEG and covert flattened tracks into single interemediate file which is then trimmed by the original clip timing. https://github.com/ynput/ayon-core/blob/develop/client/ayon_core/plugins/publish/extract_otio_audio_tracks.py

So I am assuming that main issue here was that we were ignoring audio tracks during our custom OTIO exporter.

Reply:
Awesome, thanks for confirming.